tests/Issues/Issue6689/config/configured_rule.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Php80\Rector\If_\NullsafeOperatorRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->set(NullsafeOperatorRector::class);
```

Comment:
for only single rule, you can move the fixture to the rules-tests/ instead at https://github.com/rectorphp/rector-src/tree/main/rules-tests/Php80/Rector/If_/NullsafeOperatorRector/Fixture

Reply:
done